source/solvers/gls_sharp_navier_stokes.cc

```diff
@@ -634,6 +635,12 @@ GLSSharpNavierStokesSolver<dim>::postprocess_fd(bool firstIter)
       this->write_output_results(this->present_solution);
     }
 
+  bool enable =
+    this->simulation_parameters.analytical_solution->calculate_error();
+  this->simulation_parameters.analytical_solution->set_enable(false);
+  NavierStokesBase<dim, TrilinosWrappers::MPI::Vector, IndexSet>::
+    postprocess_fd(firstIter);
```

Comment:
What is this? You do this to use the postprocess_fd without having it calculate error? 
Could you just not overload the calculate_L2_error and that would work? or does it not because the function is called from the base class?

Reply:
It is not possible to have calculate_L2_error virtual since it is from a base class